### PR TITLE
api: add start_analysis command

### LINKED ISSUE #49: cli: add start workflow command
- [x] https://github.com/reanahub/reana-workflow-controller/issues/48: add start workflow endpoint
- [x] https://github.com/reanahub/reana-server/issues/22: add get analysis output endpoint
- [x] Implement CLI according to #22, as an example PR https://github.com/reanahub/reana-client/pull/42 which implements the call to the new status endpoint.
  - [x] Copy last `server` OpenAPI spec.
  - [x] Use bravado client to make the call to `reana-server` and connect with click application [here](https://github.com/reanahub/reana-client/blob/master/reana_client/cli/workflow.py#L181).
  - [x] Test using `reana-client create workflow -f <reana_spec_file>` save `analysis_uuid` and then call `reana-client workflow start --workflow <analysis_uuid>`

### PR DESCRIPTION
**Pull request by anton-khodak**
closes #49

### CODE REVIEW

**Comment by anton-khodak** [comment]
There is a problem with reana-server API client returning 
```
None is not of type u'string'

Failed validating u'type' in schema[u'properties'][u'status']:
    {u'type': u'string'}

On instance[u'status']:
    None
```
however, the request succeeds and the workflow starts, so there must be some tiny issue with the schema. @diegodelemos any quick thoughts on that?

File: reana_client/cli/workflow.py [hunk 1]
```diff
@@ -198,15 +198,16 @@ def workflow_start(ctx, user, organization, workflow):
...
-    #     logging.debug(str(e))
+    try:
+        logging.info('Connecting to {0}'.format(ctx.obj.client.server_url))
+        response = ctx.obj.client.start_analysis(user,
+                                                 organization,
+                                                 workflow)
+        click.echo(response)
+        click.echo('Workflow `{}` has been started.'.format(workflow_name))
+
+    except Exception as e:
```
Comment (diegodelemos) [inline reana_client/cli/workflow.py]: Regarding exceptions https://github.com/reanahub/reana-server/issues/18

File: reana_client/api/client.py [hunk 2]
```diff
@@ -127,6 +127,26 @@ def create_workflow(self, user, organization, reana_spec):
...
+                                  status='start').result()
+            if http_response.status_code == 200:
+                return response
+            else:
+                raise Exception(
+                    "Expected status code 200 but replied with "
+                    "{status_code}".format(
+                        status_code=http_response.status_code))
+
+        except Exception as e:
```
Comment (diegodelemos) [inline reana_client/api/client.py]: Regarding exceptions https://github.com/reanahub/reana-server/issues/18

Comment (anton-khodak) [reply reana_client/api/client.py]: I suppose there is no need for additional status code handling here, exceptions are delivered to the client as they come from reana-server

**Comment by anton-khodak** [comment]
@diegodelemos PRs for #49 are ready for reviewing

**Comment by diegodelemos** [comment]
Could you please squash the commits? otherwise it is ready to :shipit:

**Pull request merged**